docs/uml-class-diagrams/middleware02/production/MiddlewareCommDatabaseInterfaceImplementation/MiddlewareCommDatabaseInterfaceImplementation.puml

```diff
@@ -2,9 +2,11 @@
 
 package edu.oakland.production.middleware02 {
     class MiddlewareCommDatabaseInterfaceImplementation {
+        + MiddlewareDatabase middlewareDatabase
         __
+        + MiddlewareCommDatabaseInterfaceImplementation(MiddlewareDatabase middlewareDatabase)
```

Comment:
This connects to the `DatabaseCommInterface`, right? If so, shouldn't it be as follows:
```suggestion
        + DatabaseCommInterface databaseCommInterface
        __
        + MiddlewareCommDatabaseInterfaceImplementation(DatabaseCommInterface databaseCommInterface)
```